fiwClient.ts

```diff
@@ -107,6 +107,25 @@ export const client = {
     return SingleEventSchema.parse(response.data);
   },
 
+  // fetch multiple events by their ids
+  getEventsByIds: async (ids: number[]) => {
+    // load all events in parallel
+    const responses = await Promise.all(
+      ids.map(async (id) => {
+        try {
+          const response = await axiosInstance.get(`/events/${id}`);
```

Comment:
dublicate code.  
why don't you just use the already existing getEventById()?

Reply:
Fixed